solution/java-streams-avro/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.16.0"
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:1.2.0"
     }
 }
```

Comment:
Delete lines from 1 to 8
Delete line 15
Add `id 'com.github.davidmc24.gradle.plugin.avro' version '1.2.0'` in the plugins block

Reply:
Done